analytics.js

```diff
@@ -1130,6 +1130,48 @@ class Analytics {
     processDataInAnalyticsArray(this);
   }
 
+  /**
+   * Call control pane to get client configs
+   *
+   * @param {*} writeKey
+   * @memberof Analytics
+   */
+  load(writeKey, serverUrl, options) {
+    logger.debug("inside load ");
+    // check if the below features are available in the browser or not
+    // If not present dynamically load from the polyfill cdn
+    if (
+      !String.prototype.endsWith ||
+      !String.prototype.startsWith ||
+      !String.prototype.includes ||
+      !Array.prototype.find ||
+      !Array.prototype.includes ||
+      !Promise ||
+      !Object.entries
+    ) {
+      if (!window.hasOwnProperty("polyfill")) {
```

Comment:
Is this still relevant?

Reply:
If it is not relevant, why are we using the same in the periodic checking logic of the polyfills load?

Reply:
After loading the polyfill we need to clear the interval and proceed. The checking is to make sure the loading is done.

Reply:
If `!String.prototype.endsWith ||
      !String.prototype.startsWith ||
      !String.prototype.includes ||` these are not preset that means polyfill is not present.